### PR TITLE
Fortran bindings are not shipped on sbsa

### PR DESCRIPTION
**Pull request by adibbley**
xref https://github.com/conda-forge/libcublas-feedstock/pull/6#issuecomment-1498234725

cc @leofang 

Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Review by leofang — APPROVED** [review]
LGTM. btw Alex you have write access and can push to the [bot's branch](https://github.com/regro-cf-autotick-bot/libcublas-feedstock/tree/bot-pr_arch_h50f6e9) directly 🙂 So, we can
- Apply this patch directly in #6, or
- Merge this now (no need to wait for the CI), and the bot will send another PR shortly

I am fine either way.

**Comment by adibbley** [comment]
Ah good to know! I pushed this change to the bot branch to hopefully keep things in order over there

https://github.com/conda-forge/libcublas-feedstock/pull/6/commits/f74feca23d773bab9b0044efcf970019fa823a40

**Pull request closed without merging**
